napalm_nxos/nxos.py

```diff
@@ -465,19 +465,30 @@ def get_ntp_stats(self):
         return ntp_stats
 
 
-    def get_interfaces_ip(self):
```

Comment:
Hm, odd, the diff doesn't make sense, will fix and repush for clarity even though this is minor.